gradle-plugins/build.gradle.kts

```diff
@@ -86,3 +103,49 @@ tasks {
     enabled = !version.toString().contains("SNAPSHOT")
   }
 }
+
+afterEvaluate {
+  publishing {
+    publications {
+      named<MavenPublication>("pluginMaven") {
```

Comment:
does `named` imply that this task already exists?

Reply:
Yeah Gradle plugin plugin creates it